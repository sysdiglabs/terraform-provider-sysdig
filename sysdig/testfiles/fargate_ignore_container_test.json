[
    {
        "name": "test",
        "image": "test_image:latest",
        "entryPoint": [
            "/bin/test"
        ]
    },
    {
        "name": "other",
        "image": "other_image:latest",
        "entryPoint": [
            "/bin/other"
        ]
    },
    {
        "name": "another",
        "image": "another_image:latest",
        "entryPoint": [
            "/bin/another"
        ]
    }
]