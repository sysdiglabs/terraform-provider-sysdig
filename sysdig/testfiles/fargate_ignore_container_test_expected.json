[
    {
        "Name": "test",
        "Image": "test_image:latest",
        "EntryPoint": [
            "/opt/draios/bin/instrument"
        ],
        "Command": [
            "/bin/test"
        ],
        "Environment": [
            {
                "Name": "SYSDIG_ORCHESTRATOR_PORT",
                "Value": "orchestrator_port"
            },
            {
                "Name": "SYSDIG_COLLECTOR",
                "Value": "collector_host"
            },
            {
                "Name": "SYSDIG_COLLECTOR_PORT",
                "Value": "collector_port"
            },
            {
                "Name": "SYSDIG_ACCESS_KEY",
                "Value": "sysdig_access_key"
            },
            {
                "Name": "SYSDIG_LOGGING",
                "Value": "sysdig_logging"
            },
            {
                "Name": "SYSDIG_PRIORITY",
                "Value": "priority"
            },
            {
                "Name": "SYSDIG_SIDECAR",
                "Value": ""
            },
            {
                "Name": "SYSDIG_ORCHESTRATOR",
                "Value": "orchestrator_host"
            }
        ],
        "LinuxParameters": {
            "Capabilities": {
                "Add": [
                    "SYS_PTRACE"
                ]
            }
        },
        "VolumesFrom": [
            {
                "ReadOnly": true,
                "SourceContainer": "SysdigInstrumentation"
            }
        ]
    },
    {
        "Name": "other",
        "Image": "other_image:latest",
        "EntryPoint": [
            "/bin/other"
        ]
    },
    {
        "Name": "another",
        "Image": "another_image:latest",
        "EntryPoint": [
            "/bin/another"
        ]
    },
    {
        "EntryPoint": [
            "/opt/draios/bin/logwriter"
        ],
        "Image": "workload_agent_image",
        "Name": "SysdigInstrumentation",
        "RepositoryCredentials": {
            "CredentialsParameter": "image_auth_secret"
        },
        "Environment": [
            {
                "Name": "SYSDIG_ORCHESTRATOR_PORT",
                "Value": "orchestrator_port"
            },
            {
                "Name": "SYSDIG_COLLECTOR",
                "Value": "collector_host"
            },
            {
                "Name": "SYSDIG_COLLECTOR_PORT",
                "Value": "collector_port"
            },
            {
                "Name": "SYSDIG_ACCESS_KEY",
                "Value": "sysdig_access_key"
            },
            {
                "Name": "SYSDIG_LOGGING",
                "Value": "sysdig_logging"
            },
            {
                "Name": "SYSDIG_PRIORITY",
                "Value": "priority"
            },
            {
                "Name": "SYSDIG_SIDECAR",
                "Value": ""
            },
            {
                "Name": "SYSDIG_ORCHESTRATOR",
                "Value": "orchestrator_host"
            }
        ]
    }
]